eva/parser/drop_statement.py

```diff
@@ -15,32 +15,28 @@
 from typing import List
 
 from eva.parser.statement import AbstractStatement
-from eva.parser.table_ref import TableRef
+from eva.parser.table_ref import TableInfo
 from eva.parser.types import StatementType
 
 
 class DropTableStatement(AbstractStatement):
-    """Drop Table Statement constructed after parsing the input query
+    """Drop Table Statement constructed after parsing the input query"""
 
-    Attributes:
-        TableRef: table reference in the drop table statement
-    """
-
-    def __init__(self, table_refs: List[TableRef], if_exists: bool):
+    def __init__(self, table_infos: List[TableInfo], if_exists: bool):
         super().__init__(StatementType.DROP)
-        self._table_refs = table_refs
+        self._table_infos = table_infos
```

Comment:
Just trying to understand the difference here. Why is this changed to `TableInfo` from `TableRef`?

Reply:
The idea is that `drop` will always be on a table, so it is unnecessary to use the abstraction `TableRef` which can be a join, subquery etc